src/shared/format-table.h

```diff
@@ -27,6 +27,8 @@ typedef enum TableDataType {
         TABLE_UINT64,
         TABLE_PERCENT,
         TABLE_IFINDEX,
+        TABLE_IN_ADDR,  /* Takes union in_addr_union */
```

Comment:
hmm, the comment is a bit misleading, after all the union consists of just a struct in_addr and a struct in6_addr, i.e. passing a "struct in_addr" here is totally OK too, I'd say...

Reply:
maybe change to "Takes a union in_addr_union (or a struct in_addr)"